pkg/pb/inspector.proto

```diff
@@ -110,6 +112,17 @@ message LookupNodeResponse {
   node.NodeMetadata meta = 2;
 }
 
+message NodeInfoRequest {
+  bytes id = 1 [(gogoproto.customtype) = "NodeID", (gogoproto.nullable) = false];
```

Comment:
I'm confused. Why is this here?

Reply:
ah, wait this is the kademlia thing.

Reply:
Initially it was only the address in the `NodeInfoRequest`. Then, when you asked to remove the ID from the response, I added the ID in the `NodeInfoRequest`, just for the sake for looking like other similar requests - `PingRequest` and `FetchPeerIdentity`.

I removed it again in the latest commit, because it does not makes sense to me. Having only the address is enough to complete the `NodeInfoRequest`. This may be helpful for other use cases where the caller knows only the address of the bootstrap node or the satellite and wants to find its ID using the `NodeInfoRequest`.